src/main/scala/com/databricks/labs/overwatch/api/ApiCallV2.scala

```diff
@@ -647,6 +654,11 @@ class ApiCallV2(apiEnv: ApiEnv) extends SparkSessionWrapper {
     }
     try {
       executeHelper()
+      if (!apiMeta.storeInTempLocation) {
+        //write to temp success location for success
+        //write to temp error location for error
+      }
+      this// persistEnriched
```

Comment:
persistEnriched func call should be inside the if statement. And why would you put the write to temp / error inside the !storeInTemp -- that should only be done for when there is storeInTemp otherwise we should just persist the enriched results from inside the if statement